README.md

```diff
@@ -1,2 +1,27 @@
 # click-apps-for-openwrt
-OpenWrt based click applications
+
+###Overview
+
```

Comment:
There has to be some build instructions as well. such as to add the feed from openwrt-feeds and then select in the menuconfig etc and then build openwrt.